modules/ROOT/pages/glossary.adoc

```diff
@@ -15,69 +15,72 @@ endif::[]
 [[secta]]
 *A*
 
-Activity:: Role-based document transaction tracking (for send and receive) based on rules, actions, and functions.
+Acknowledgment:: Sent as a response to a B2B transmission. A TA1 acknowledgment serves as a receipt that a transmission was received. A functional acknowledgment includes details of the validation process.
```

Comment:
Need to spell out TA1 and provide more context. TA1 can be associated with multiple sources such as X12, HIPAA, or EDISS. Please state what source this TA1 utilizes.

Reply:
This is a great point. I will make this clear in the body of the doc, since we will not use a glossary for this product.